OPT_mission.Tanoa/sectorcontrol/functions/fnc_initplayereh.sqf

```diff
@@ -33,13 +33,13 @@ if (EGVAR(training,on)) then {
         // is there a flag that should be moved?
         _flag = objNull;
         {
-            if (_x getVariable [QGVAR(isFlagMovable), false]) exitWith {_flag = _x};
+            if ((_x select 0) getVariable [QGVAR(isFlagMovable), false]) exitWith {_flag = (_x select 0)};
```

Comment:
Hier könnte man sich das `_x select 0` auch sparen, wenn man die Flaggen nicht jeweils nochmal in ein extra Array verpackt :)

Reply:
https://github.com/OperationPandoraTrigger/OPT-Mission/pull/101#discussion_r384068015